src/views/ConsentContinuationOut.vue

```diff
@@ -88,6 +88,26 @@
               </div>
             </section>
 
+            <!--  Jurisdiction Information -->
+            <section>
+              <header>
+                <h2>Jurisdiction Information</h2>
+              </header>
+              <div :class="{ 'invalid-foreign-jurisdiction': !foreignJurisdictionValid && showErrors }"
+                id="foreign-jurisdiction-section">
+                <v-card flat class="pt-6 px-4">
+                  <ForeignJurisdiction
+                    ref="foreignJurisdictionRef"
+                    :validateForm="showErrors"
+                    :draftCountry.sync="draftCountry"
+                    :draftRegion.sync="draftRegion"
+                    @update:country="selectedCountry=$event"
+                    @update:region="selectedRegion=$event"
```

Comment:
I think the `sync` modifiers above are used to keep the properties updated (kind of like a v-model), so you shouldn't need the `@update` event handlers.

Or vice-versa. (I think you only need one or the other.)

Reply:
removed `sync`